//Error: Statement 'continue' can only appear inside a loop!
//3
fn hello()->void {
  while 10: //<- error
    continue;
  continue; //<- error
  break; //<- error
}
